.github/workflows/docs-links.yaml

```diff
@@ -17,7 +17,8 @@ jobs:
         run: sudo pip install LinkChecker
 
       - name: Run linkchecker
-        run: linkchecker --threads 2 --ignore-url "https://assets.ubuntu.com$" --ignore-url "https*://127.0.0.1.*" --ignore-url ".*.xip.io/.*" --check-extern --no-warnings https://microk8s.io/docs
+        run: linkchecker --threads 2 --ignore-url "https://assets.ubuntu.com$" --ignore-url "https*://127.0.0.1.*" --ignore-url ".*.xip.io.*" --ignore-url ".*.nip.io.*" --check-extern --no-warnings https://microk8s.io/docs
```

Comment:
Can you remove xip.io now as it was changed to nip.io?

Reply:
I guess we can. there is a possibility it may change back again, but I guess we can always edit this again if that happens.